pytest/tests/mocknet/README.md

```diff
@@ -1,12 +1,18 @@
 Mirror transactions from a given network into a custom mocktest network and add load
 
-1. Setup a custom mocknet network following the README in the `provisioning/terraform/network/mocknet/mirror/` directory of the [near-ops](https://github.com/near/near-ops) repo.
-- you'll need the `unique_id`, `chain_id`, and `start_height` from this setup when running the mirror.py test script in 2.
+1. Setup a custom mocknet network following the [README](https://github.com/Near-One/infra-ops/blob/main/provisioning/terraform/infra/network/mocknet/mirror/README) in the `provisioning/terraform/infra/network/mocknet/mirror/` directory of the [Near-One/infra-ops repository](https://github.com/Near-One/infra-ops).
+    - An example setup command should look like the following: `terraform apply -var="unique_id=stateless" -var="chain_id=mainnet" -var="start_height=116991260" -var="size=small"`
+
+    - Use the same values of `unique_id`, `chain_id`, and `start_height` from this setup when running the mirror.py commands below. 
+
 2. Run `python3 tests/mocknet/mirror.py --chain-id {chain_id} --start-height {start_height} --unique-id {unique_id} init-neard-runner`, replacing the `{}`s with appropriate values from the `nearcore/pytest` directory. This starts a helper program on each node that will be in charge of the test state and neard process.
+
 3. Run `python3 tests/mocknet/mirror.py --chain-id {chain_id} --start-height {start_height} --unique-id {unique_id} new-test`. This will take a few hours.
+
 4. Run `python3 tests/mocknet/mirror.py --chain-id {chain_id} --start-height {start_height} --unique-id {unique_id} start-traffic` replacing the `{}`s with appropriate values
+
 5. Monitoring
-- See metrics on grafana mocknet https://grafana.near.org/d/jHbiNgSnz/mocknet?orgId=1&refresh=30s&var-chain_id=All&var-node_id=.*unique_id.*&var-account_id=All replacing the "unique_id" with the value from earlier
+    - See metrics on grafana mocknet https://grafana.near.org/d/jHbiNgSnz/mocknet?orgId=1&refresh=30s&var-chain_id=All&var-node_id=.*unique_id.*&var-account_id=All replacing the "unique_id" with the value from earlier
```

Comment:
We can replace it with https://nearone.grafana.net/ or https://nearinc.grafana.net/

Reply:
Done. Will keep nearinc now, as mocknet dashboard is not migrated there.